rest/api.go

```diff
@@ -159,6 +159,8 @@ type API interface {
 	// responseSerializer returns a ResponseSerializer for the given format type. If the
 	// format is not implemented, the returned serializer will be nil and the error set.
 	responseSerializer(string) (ResponseSerializer, error)
+
+	Router() *mux.Router
```

Comment:
It's exposed here solely to make it available [in the requestHandler methods](https://github.com/aaronkavlie-wf/go-rest/blob/build_url/rest/handler.go#L134). Is there another way to get from here to there without exposing the `mux.Router`?